native/src/widget/text_input.rs

```diff
@@ -683,6 +683,30 @@ impl State {
     pub fn cursor(&self) -> Cursor {
         self.cursor
     }
+
+    /// Moves the [`Cursor`] of the [`TextInput`] to the front of the input text.
+    ///
+    /// [`Cursor`]: struct.Cursor.html
+    /// [`TextInput`]: struct.TextInput.html
+    pub fn move_cursor_to_front(&mut self) {
```

Comment:
Given that this could be achieved with `move_cursor_to(0)`, is this method necessary?

Reply:
I implemented the front/end methods because they are self documenting.